src/backend/marsha/settings.py

```diff
@@ -33,6 +33,7 @@ class Base(Configuration):
     # Static files (CSS, JavaScript, Images)
     STATICFILES_DIRS = (os.path.join(BASE_DIR, "static"),)
     STATIC_URL = "/static/"
+    STATIC_BASE_URL = f"{STATIC_URL}js/"
```

Comment:
Ok I'm going to bikeshed you a little bit.

in Django there is a setting named STATIC_URL.
So STATIC_BASE_URL is a bad name for this as a setting.

I propose to not put the /js inside it and call it ABSOLUTE_STATIC_URL...  After all it could be used by someone else in the backend...

Reply:
Done